framework/test/mocha/functional/ws/transport/blocks.js

```diff
@@ -180,10 +182,10 @@ describe('WS transport blocks', () => {
 			} catch (err) {
 				__testContext.debug(
 					'> Error / Response:'.grey,
-					JSON.stringify(err),
+					JSON.stringify(err.errorResponse),
```

Comment:
why not checking err.message here?

Reply:
Because we are only capturing the message here, whereas in `errorResponse` we are actually capturing the error or string whatever is coming from the response from the peer.